platform-configs/protectli-vp4650.robot

```diff
@@ -178,6 +182,9 @@ ${RESET_TO_DEFAULTS_SUPPORT}=                       ${TRUE}
 ${MEMORY_PROFILE_SUPPORT}=                          ${FALSE}
 ${DEFAULT_POWER_STATE_AFTER_FAIL}=                  Powered Off
 ${ESP_SCANNING_SUPPORT}=                            ${FALSE}
+${UUID_SETTABLE}=                                   ${TRUE}
+${SERIAL_NUMBER_SETTABLE}=                          ${TRUE}
+${BOOTSPLASH_SETTABLE}=                             ${TRUE}
```

Comment:
We already have flag for logo customization, right?

Reply:
Reworked to use `CUSTOM_LOGO_SUPPORT` [here](https://github.com/Dasharo/open-source-firmware-validation/pull/139/commits/ad98d1323929b34a1e69ec7c1e73bc6ecd7bed42#diff-7ee5e21e59cd627383812396fcf45a6e7ecb3e2f096ab5ae5203a0c6bd75989fR74) and removed the unused flag